spec/features/user_updates_review_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+feature 'user updates review', %{
+  As a user
+  I want to update a review
+  So that I can correct details
+} do
+
+  # Acceptance Criteria:
+  # User must be logged in
+  # User can only edit his/her own review
+
+  scenario 'user wants to edit his/her review' do
+    user = FactoryGirl.create(:user)
+    category = FactoryGirl.create(:category)
+    item = FactoryGirl.create(:item, user: user, category: category)
+    review = FactoryGirl.create(:review, user: user, item: item)
```

Comment:
You could `review = FactoryGirl.create(:review)` and use associated objects `(review.item, review.item.user, etc) to save some lines of code, here.